src/pages/guides/using/creating_actions.md

```diff
@@ -186,9 +188,70 @@ function main(params) {
 
 exports.main = main;
 ```
-
 You can also set cookies or cache control headers, perform a HTTP redirect, and so forth.
 
+### Unsuccessful response 
+
+On failed web action invocations, the error code and message should be wrapped in an `error` object, as this would allow the system to interpret the response as an `applicationError`. 
+
+```javascript
+function main(params) {
+    try {
+        throw new Error("Boom!")
+    } catch (err) {
+        return {
+            error: {
+                statusCode: 500,
+                body: {
+                    payload: `Something went wrong: ${err}`
+                }
+            }
+        }
+    }
+}
+
+exports.main = main;
+```
+
+### Dealing with errors in general
+It is extremely important to handle errors in your action code. It is actually impossible to recover from an unhandled async error in Node.js. Please refer to the Node.js documentation related to [uncaughtExceptions](https://nodejs.org/api/process.html#event-uncaughtexception).
+
+In the event of an unhandled async errors, the action will be terminated and the container running that action will be destroyed. This means that all in flight activations will be failed and, the next invocation of the action will incur the overhead of creating a new container.
```

Comment:
Did we want to mention "in the case of using container concurrency (enabled by default) ... ? 🤷